src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -343,6 +347,9 @@ export default {
 				|| genderFilterApplied
 				|| sectorFilterApplied
 				|| themeFilterApplied;
+		},
+		themeNames() {
+			return this.allFacets?.themeNames;
```

Comment:
Looks like there probably needs to be a `?? []` here so that selecting based on prop in the child component doesn't potentially encounter an exception. Edge case, but just in case.